dot/rpc/subscription/listeners_test.go

```diff
@@ -70,7 +70,7 @@ func TestStorageObserver_Update(t *testing.T) {
 }
 
 func TestBlockListener_Listen(t *testing.T) {
-	notifyChan := make(chan *types.Block)
+	notifyChan := make(chan *types.Block, 100)
```

Comment:
Can you make the value `100` constant and all packages should use it?
It will make it easy to configure.

Reply:
I've added const DEFAULT_BUFFER_SIZE, and removed the buffers from the channels that are used in tests.

Reply:
This one could be unbuffered?